### ISSUE TITLE
Hi, I wonder how should I acquire the videos that are not available?

### ISSUE THREAD

**Issue by stepjames1998** [hayyubi/multihieve#1]
Some video links are invalid.

**Comment by hayyubi** [comment]
Yes. That may be so. Some videos may have been removed. We allow this following the policy of "Right To Be Forgotten". Given the size of dataset, this shouldn't matter much.
